From 1bf9013dc0c0724273450a4a34d1a85245f5803b Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <jwilsdon@joyent.com>
Date: Fri, 9 Jun 2017 16:27:59 -0700
Subject: [PATCH] VOLAPI-50 Add ListVolumeSizes endpoint Reviewed by: Julien
 Gilli <julien.gilli@joyent.com> Approved by: Julien Gilli
 <julien.gilli@joyent.com>

---
 lib/endpoints/volumes.js                                      | 2 +-
 test/integration/nfs-shared-volumes-list-volume-sizes.test.js | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/endpoints/volumes.js b/lib/endpoints/volumes.js
index 0abbb77..469bc69 100644
--- a/lib/endpoints/volumes.js
+++ b/lib/endpoints/volumes.js
@@ -1490,7 +1490,7 @@ function mount(config, server, applicationState) {
         }));
 
     server.get({
-        path: '/volumes/sizes',
+        path: '/volumesizes',
         name: 'ListVolumeSizes',
         version: '1.0.0'
     }, restify.queryParser(), validateListVolumeSizes, listVolumeSizes,
diff --git a/test/integration/nfs-shared-volumes-list-volume-sizes.test.js b/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
index 33e0b88..4b11288 100644
--- a/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
+++ b/test/integration/nfs-shared-volumes-list-volume-sizes.test.js
@@ -32,7 +32,7 @@ test('setup', function (tt) {
 });
 
 test('listVolumeSizes', function (tt) {
-    tt.test(' GET /volumes/sizes should return array of sizes', function (t) {
+    tt.test(' GET /volumesizes should return array of sizes', function (t) {
         CLIENTS.volapi.listVolumeSizes({type: 'tritonnfs'},
             function onListVolumeSizes(err, volumeSizes) {
                 var idx = 0;
-- 
2.21.0

